source/educators/how-tos/content-tagging-how-tos/build_taxonomy_using_template.rst

```diff
@@ -0,0 +1,95 @@
+Build a taxonomy using a template
+#################################
+
+Building a taxonomy using a template
+************************************
+
+Tags that can be applied to content are organized into hierarchical taxonomies. Currently, the only way to create or modify a taxonomy is by uploading a CSV or JSON file.
+
+The simplest type of taxonomy is a flat taxonomy that doesn’t have any hierarchy - see `“How-to: Create a flat taxonomy by uploading a CSV” <Create_flat_taxonomy_by_uploading_CSV.rst>`_ for a simpler procedure to create such a flat taxonomy. For the purposes of this guide, we’ll assume you want to create a taxonomy of cities, like this:
+
+* United States
+
+  * California
+
+    * Los Angeles
+
+  * New York
+
+    * New York City
+
+* Mexico
+
+  * Mexico City
+
+Steps to build a taxonomy using a template
+==========================================
+  
+#. First, review your proposed taxonomy. Note that each tag must have a unique name (across all levels of the hierarchy). So, for example, we could not have “New York” as both a second-level (state) tag and third-level (city) tag - we have to rename one of them (e.g. “New York City” in this case)
+
+#. Navigate to Studio Home > Taxonomies. You should see Download Template in the top right. Click on that button, then select “CSV template”.
+
+   .. image:: /_images/educator_how_tos/ctag_taxonomy_template_step2.png
+      :alt: Screenshot clicking Download Template button.
+
+#. A file will download automatically. Open it using any spreadsheet application (Excel, Numbers, Google Sheets, etc.).
+
+#. It should look like this:
+
+   .. image:: /_images/educator_how_tos/ctag_taxonomy_template_step4.png
+      :alt: Image showing a fully-populated spreadsheet with four columns labeled: id, value, parent_id, and comments.
+
+#. Select and delete all of the entries except the first row, which contains the headings.
+
+   .. image:: /_images/educator_how_tos/ctag_taxonomy_template_step5.png
+      :alt: Image showing an empty spreadsheet with four columns labeled: id, value, parent_id, and comments.
+
+Create the first tag
+********************
+
+#. Now, you can start to enter the tags that make up the taxonomy. The order doesn’t really matter, except that any tags which come “below” another tag should also be below/after it in the spreadsheet. For our example taxonomy, we shouldn’t put “California” before “United States”, because the “California” tag is below (is a child of) the “United States” tag. 
```

Comment:
So, because you put a header, your numbers start over from 1 after each heading.

<img width="310" alt="image" src="https://github.com/user-attachments/assets/c5019caa-8eeb-4689-acda-02f16169b606">

Is that what you want? I'm happy to discuss options (probably I'd suggest using bullets instead of numbered list, or have the headers actually be the first item and the options be subbullets eg,


1. Review your proposed taxonomy

    #. step 1

    #. step 2

#. Create the first tag

    #. step 3

#. Creating "child" tags

    #. step 4

Reply:
I like your suggestions -  I'll try using bullets instead of a numbered list - I'll try that and see how it looks.